perun-contract/src/Perun/Offchain/ChannelTxPool/Types.hs

```diff
@@ -0,0 +1,89 @@
+{-# LANGUAGE FunctionalDependencies #-}
+
+module Perun.Offchain.ChannelTxPool.Types where
+
+import Control.Lens.TH
+import Ledger (TxOutRef)
+import Perun.Onchain
+import Plutus.ChainIndex hiding (txFromTxId)
+
+-- | ChannelTxPool is a pool of ChannelTxs for a specific channel id.
+-- Following invariants hold:
+--  1. All TXs contained are ONLY relevant for a single channel id.
```

Comment:
This is either incomplete or misleading (if there is only one invariant, we should not use `1.`)

Reply:
Yes, residual comment from before our `ChannelToken` adaption.